test/extract_method_baseline/SimpleMethods.java

```diff
@@ -0,0 +1,38 @@
+class SimpleMethods {
```

Comment:
why we have so a simple test? could we use more sophisticated methods? Maybe we can take it from the paper?

Reply:
Then it will be more close to integration tests. We better keep unit tests simple for easy debug. If you want some integration tests we need to understand what exactly they should solve? Is it just fixing behavior on some production code? Or something more?

Reply:
I will add more complex tests later when complex statements will be supported.